staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -59,3 +59,54 @@ type InTreePlugin interface {
 	// RepairVolumeHandle generates a correct volume handle based on node ID information.
 	RepairVolumeHandle(volumeHandle, nodeID string) (string, error)
 }
+
+// convertZones converts existing zone labels to NodeAffinity.
+func convertZones(pv *v1.PersistentVolume, topologyKey string) error {
+	zones := []string{}
+	for k, v := range pv.ObjectMeta.Labels {
+		if k == v1.LabelZoneFailureDomain {
+			zones = append(zones, v)
```

Comment:
I think this won't handle Regional PD zone syntax, which is:
failure-domain.beta.kubernetes.io/zone: “europe-west1-b__europe-west1-c”

Reply:
Resolving in favor of https://github.com/kubernetes/kubernetes/pull/83394/files#r338330785